data/sql/updates/pending_db_world/rev_1687513981074967100.sql

```diff
@@ -0,0 +1,58 @@
+-- Hearts of the Pure Rp -------------
+
+-- The orientation of the initial NPC This is roughly towards the coordinates that need to be sniffed 
+-- UPDATE `creature` SET `orientation`=4.41048 WHERE `guid`=41833;
+
+-- 3 emoticons when preparing for a task
+DELETE FROM `quest_details` WHERE `ID`=1476;
+-- EMOTE_ONESHOT_QUESTION && EMOTE_ONESHOT_LAUGH && EMOTE_ONESHOT_TALK
+INSERT INTO `quest_details` (`ID`, `Emote1`, `Emote2`, `Emote3`, `Emote4`, `EmoteDelay1`, `EmoteDelay2`, `EmoteDelay3`, `EmoteDelay4`, `VerifiedBuild`) VALUES
+(1476, 6, 11, 1, 0, 0, 0, 0, 0, 0);
```

Comment:
quest_details are sniffed so you should leave it as it was before. It's common to see emotes from these overlap with RP if players are too fast